docs/manifest.json

```diff
@@ -165,6 +165,15 @@
       ]
     },
     {
+      "id": "speech",
+      "name": "@google-cloud/speech",
+      "defaultService": "speech",
+      "versions": [
+        "0.1.0",
```

Comment:
Since this isn't actually released, I think you should wait to include the release docs until we actually cut a release.